internal/database/config.go

```diff
@@ -25,7 +25,7 @@ type Config struct {
 	Port               string        `env:"DB_PORT, default=5432"`
 	SSLMode            string        `env:"DB_SSLMODE, default=require"`
 	ConnectionTimeout  int           `env:"DB_CONNECT_TIMEOUT"`
-	Password           string        `env:"DB_PASSWORD"`
+	Password           string        `env:"DB_PASSWORD" json:"-"`
```

Comment:
so, the json:"-" is interpreted by the logging infrastructure?

Reply:
it's fine if so - but would deserving of a comment and maybe a link

Reply:
Added comment.